css-four-column-layout/styles.css

```diff
@@ -10,10 +10,11 @@ body {
 
 .row {
   display: flex;
+  justify-content: center;
```

Comment:
You should move this to a separate class that you can add in addition to the `row` class.